addon/services/cloudinary.js

```diff
@@ -181,5 +181,92 @@ export default Ember.Service.extend({
     }
 
     return `${protocol}${subDomain}.${domain}${pathname}`;
+  },
+
+
+  /** @property {String[]} _concatenatedTransforms */
+  _concatenatedTransforms: computed('config.CONCATENATED_TRANSFORMS', function() {
+    // Protection from Strings
+    const concatenatedTransforms = this.get('config.CONCATENATED_TRANSFORMS') || [];
+    return Array.isArray(concatenatedTransforms) ? concatenatedTransforms : [concatenatedTransforms];
+  }),
+
+  /**
+   * @method getURL
+   * @param  {String}          [publicId]           Public id of image in Cloudinary
+   * @param  {String}          hash.format          File extension)
+   * @param  {String}          hash.cloudName
+   * @param  {(String|Number)} hash.width
+   * @param  {(String|Number)} hash.height
+   * @param  {(String|Number)} hash.version
+   * @param  {String}          hash.domain
+   * @param  {String}          hash.subDomain
+   * @param  {Boolean}         hash.cdnDistribution
+   * @param  {Boolean}         hash.secure
+   * @param  {String}          hash.type
+   * @param  {String[]}        hash.transforms
+   * @return {String}                               URL for image
+   */
+  getURL(publicId = [], hash = {}) {
```

Comment:
`publicId` not need a default parameter, and if so, so it needs to be an empty string